### PR TITLE
fix(appflowy_flutter): fix cover image overflow #1916

### LINKED ISSUE #1916: [Bug] Cover Image causes Overflow
### Bug Description

Adding an cover image in document causes an overflow.

Screenshot from 2023-03-03 18-09-48

### How to Reproduce

Run the app, using the `AF-desktop: Build All` and create a document. Add a cover image, you will be able to see the overflow.

### Expected Behavior

No Overflow, should be present.

### Operating System

Ubuntu 20.04.5 LTS

### AppFlowy Version(s)

1.0.0.1

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by hyj1204**
Fix cover image overflow issue. Fix #1916

### CODE REVIEW

**Comment by AmanNegi** [comment]
Hey there, you removed the Unconstrained Box that was causing the overflow. However as you can see the image is not fully covered across width, due to the padding provided to the editing section. As @LucasXu0 mentioned, the`UnconstrainedBox` is used intentionally.

Screenshot from 2023-03-10 07-47-07

**Comment by LucasXu0** [comment]
Hey, @hyj1204. The `UnconstrainedBox` is used intentionally. Do you know if is there any way to disable the debug label?

File: frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/cover_node_widget.dart [hunk 1]
```diff
@@ -281,12 +281,18 @@ class _CoverImageState extends State<_CoverImage> {
...
-        padding: const EdgeInsets.only(bottom: 10),
-        height: height,
-        width: screenSize.width,
-        child: coverImage,
+//OverflowBox needs to be wraped by a widget with contraints(or from its parent) first,otherwise it will occur an erorr
+    return SizedBox(
+      height: height,
+      child: OverflowBox(
+        maxWidth:
+            screenSize.width + editorState.editorStyle.padding!.horizontal,
```
[the remark this answers is not in the document]
Comment (appflowy) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/cover_node_widget.dart:289]: That's awesome!

**Comment by AmanNegi** [comment]
This seems perfect! :+1:

File: frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/cover_node_widget.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (LucasXu0) [reply frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/cover/cover_node_widget.dart:289]: Hey, @hyj1204. Great. By the way, I think just assigning the `screenSize.width` to `maxWidth` is fine. Or the image will be overstretched.

Screenshot 2023-03-11 at 21 10 12

Screenshot 2023-03-11 at 21 10 27

**Pull request merged**